### DISCUSSION TITLE
Keep plugin window open on server?

### DISCUSSION THREAD

**Discussion by woodentoe** [apohl79/audiogridder#703] [Ideas]
Is it possible to make the plugins on the Server machine stay open when I close the plugin in my daw?

I'm using Audiogridder to set up an old machine as a meter so I need the metering plugins to stay open all the time 
I know I can set the plugin on my main machine to stay open in Pro Tools but I'd prefer not to

**Comment by CaBuffalo** [comment]
No, it's not possible to do that.

On Mon, Aug 30, 2021 at 3:09 AM woodentoe ***@***.***> wrote:

> Is it possible to make the plugins on the Server machine stay open when I
> close the plugin in my daw?
>
> I'm using Audiogridder to set up an old machine as a meter so I need the
> metering plugins to stay open all the time
> I know I can set the plugin on my main machine to stay open in Pro Tools
> but I'd prefer not to
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/703>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BZ24NEW2WN6KP4DZLT7NKEXANCNFSM5DBRJS6A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
Thats correct, this is not possible right now. I will create a feature request. This will require a couple of changes to the server code.
